docs/DeveloperGuide.md

```diff
@@ -325,7 +349,97 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
-*{More to be added}*
+**Use case (UC3) : Schedule an appointment for the patient**
+
+**MSS**
+
+1.  User requests to list persons
+2.  AddressBook shows a list of persons
+3.  User requests to schedule an appointment for a specific person in the list
+4.  AddressBook prompts User to input appointment details
+5.  User inputs appointment details 
+6.  AddressBook schedules appointment for the patient
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. The list is empty.
+
+  Use case ends.
+
+* 5a. The given appointment details are invalid.
+
+    * 5a1. AddressBook shows an error message.
+
+      Use case resumes at step 4.
+
+**Use case (UC4) : Cancel an appointment**
+
+**MSS**
+
+1.  User requests to list persons
+2.  AddressBook shows a list of persons
```

Comment:
Should we be showing a list of appointments instead?

Reply:
That's true, but if I'm not wrong for this version since there's only a "View all upcoming appointments" (aka appointments for the day) that would mean appointments made for the future can't be cancelled 😅 

Should we change the view all upcoming appointments to show all appointments? Or add a new case to show all appointments?

Reply:
I think for now, we can leave it at "View all upcoming appointments" (for all appointments, including those beyond the day). If time permits, we can add a command to filter it for the day. How does that sound?

Reply:
Sounds good! Thank, will modify it